From b8ce2063cdabc011f68d5efb65a7d01857a211ae Mon Sep 17 00:00:00 2001
From: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Date: Thu, 1 Aug 2019 14:25:33 +0200
Subject: [PATCH 361/529] ARM: dts: stm32: declare m4 system resource manager
 on STM32MP151

Define the m4_system_resources sub-node of m4_rproc.

Signed-off-by: Fabien Dessenne <fabien.dessenne@foss.st.com>
Signed-off-by: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
reviewed on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/283823
Change-Id: I13b5526b284281debd7b95f94ea897424d0efc1f
---
 arch/arm/boot/dts/stm32mp151.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

--- a/arch/arm/boot/dts/stm32mp151.dtsi
+++ b/arch/arm/boot/dts/stm32mp151.dtsi
@@ -2013,6 +2013,11 @@
 			st,syscfg-rsc-tbl = <&tamp 0x144 0xFFFFFFFF>;
 			st,syscfg-m4-state = <&tamp 0x148 0xFFFFFFFF>;
 			status = "disabled";
+
+			m4_system_resources {
+				compatible = "rproc-srm-core";
+				status = "disabled";
+			};
 		};
 	};
 };
